app/models/mixins/authentication_mixin.rb

```diff
@@ -78,8 +78,8 @@ def authentication_service_account(type = nil)
     authentication_component(type, :service_account)
   end
 
-  def required_credential_fields(_type)
-    [:userid]
+  def required_credential_fields(type)
+    type.to_s == "bearer" ? [] : [:userid]
```

Comment:
@enoodle now that I think of it shouldn't `:auth_key` be returned (required) for `"bearer"`?